### DISCUSSION TITLE
Draft Previews for nested collections

### DISCUSSION THREAD

**Discussion by rschlund** [payloadcms/payload#4973] [Q&A]
If i have a collection with a preview mode, it is no problem to enter preview as soon as the collection is saved as draft. But when one of its relations is saved as draft, switching to draft preview mode doesn't show the relation as draft, because the collection itself is not in draft mode. Is there a way to achieve this?

**Comment by evelynhathaway** [comment]
In case anyone finds this discussion, this was a duplicate of https://github.com/payloadcms/payload/discussions/3963
